resources/views/maps/markers/edit.blade.php

```diff
@@ -60,6 +61,19 @@
                                 </a>
                             </li>
                         </ul>
+                    </div>  
+                    </div>  
+
+                    <div class="pull-left">
+                        <div class="btn-group">
+                                <a role="button" tabindex="-1" class="btn btn-dynamic-delete btn-danger" data-toggle="popover"
+                                    title="{{ __('crud.delete_modal.permanent') }}"
```

Comment:
To keep the pop-out consistent in Kanka, please use `'crud.delete_modal.permanent'` as the title translation key

Reply:
Im not sure im undrestanding correctly, in the code you have highlighted `'crud.delete_modal.permanent'` is already being used as the title translation key, what im I supposed to change?

Reply:
Changes implemented